plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestHdfsParquetDataSource.java

```diff
@@ -20,36 +20,43 @@
 import io.airlift.slice.Slice;
 import io.airlift.slice.Slices;
 import io.airlift.units.DataSize;
+import io.trino.filesystem.TrinoFileSystem;
 import io.trino.parquet.ChunkReader;
 import io.trino.parquet.DiskRange;
-import io.trino.parquet.ParquetDataSourceId;
 import io.trino.parquet.ParquetReaderOptions;
 import io.trino.plugin.hive.FileFormatDataSourceStats;
-import org.apache.hadoop.fs.FSDataInputStream;
 import org.apache.hadoop.fs.PositionedReadable;
 import org.apache.hadoop.fs.Seekable;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.UUID;
 import java.util.stream.IntStream;
 
+import static io.trino.plugin.hive.HiveTestUtils.HDFS_FILE_SYSTEM_FACTORY;
+import static io.trino.plugin.hive.HiveTestUtils.SESSION;
 import static java.util.Objects.requireNonNull;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestHdfsParquetDataSource
 {
     @Test(dataProvider = "testPlanReadOrderingProvider")
     public void testPlanReadOrdering(DataSize maxBufferSize)
+            throws IOException
     {
         Slice testingInput = Slices.wrappedIntArray(IntStream.range(0, 1000).toArray());
-        HdfsParquetDataSource dataSource = new HdfsParquetDataSource(
-                new ParquetDataSourceId("test"),
-                0,
-                new FSDataInputStream(new TestingSliceInputStream(testingInput.getInput())),
```

Comment:
TestingSliceInputStream seems to be unused now, it can be deleted

Reply:
Good point, I haven't noticed that